content/1_docs/1_guide/20_kirby-and-privacy/guide.txt

```diff
@@ -0,0 +1,63 @@
+Title: Kirby & Privacy
+
+----
+
+Description: Things you need to know about Kirby in terms of privacy and GDPR compliance.
+
+
+----
+
+Text:
+
+<info>
+The following information might be incomplete. Your site may collect or process personal data in other ways.
+
+We cannot give any legal advice what this means for you and your sites. If you are unsure what you need to do to comply with GDPR in particular or any other national privacy laws in your country, please consult a lawyer.
```

Comment:
Maybe "what the following information means for you and your sites" so that the disclaimer applies to everything.